src/main/java/uk/gov/hmcts/reform/amapi/controllers/AccessManagementController.java

```diff
@@ -70,7 +70,7 @@
                                      @RequestHeader(name = "Caller", defaultValue = "Anonymous") String caller) {
         try {
             MDC.put("caller", caller);
-            am.revokeResourceAccess(request);
+            accessManagementService.revokeResourceAccess(request);
             return new ResponseEntity<>(NO_CONTENT);
         } finally {
             MDC.clear();
```

Comment:
do we really need MDC to set the caller on the apis? can you investigate this?

Reply:
Removed